Unshaky/pl.lproj/Counter.strings

```diff
@@ -3,19 +3,19 @@
 "8E2-Qi-cCa.title" = "-";
 
 /* Class = "NSWindow"; title = "Statistics"; ObjectID = "QEc-T5-kPG"; */
-"QEc-T5-kPG.title" = "Statistics";
+"QEc-T5-kPG.title" = "Statystyki";
 
 /* Class = "NSTextFieldCell"; title = "-"; ObjectID = "UsH-UE-4NM"; */
 "UsH-UE-4NM.title" = "-";
 
 /* Class = "NSTableColumn"; headerCell.title = "Key"; ObjectID = "Y81-5k-LdE"; */
-"Y81-5k-LdE.headerCell.title" = "Key";
+"Y81-5k-LdE.headerCell.title" = "Klawisz";
 
 /* Class = "NSTextFieldCell"; title = "-"; ObjectID = "Yte-dc-JdH"; */
 "Yte-dc-JdH.title" = "-";
 
 /* Class = "NSTableColumn"; headerCell.title = "Dismissed"; ObjectID = "m7I-RF-TDD"; */
-"m7I-RF-TDD.headerCell.title" = "Dismissed";
+"m7I-RF-TDD.headerCell.title" = "Uniknięto";
```

Comment:
Maybe `Odkliknięto` or `Odrzucono` might be a better one?

Reply:
Makes sense, will do